pkg/menu/menu.go

```diff
@@ -396,7 +394,7 @@ func PromptMenuEntry(menuTitle string, introwords string, entries []Entry, uiEve
 	logBox.Title = "Logs:"
 	logBox.WrapText = false
 	logBox.SetRect(0, location, windowWidth, location+height)
-	logBox.Rows = make([]string, 1000)
+	logBox.Rows = make([]string, 1)
```

Comment:
You don't need a make if you are appending to a []string
https://play.golang.org/p/l8W7duYKXgc

This is the rule that "the zero value should be useful". The zero value of a []string is something you can append to, which is very handy.

Reply:
so can you quick try this without the make